ansible/roles/docker/tasks/main.yaml

```diff
@@ -0,0 +1,36 @@
+#/roles/docker/tasks
+
+- name: Add GPG Docker key
+  apt_key:
+    url: "{{ docker_apt_key }}"
+    state: present
+
+- name: Add Docker Repo
+  apt_repository:
+    repo: "{{ docker_apt_repo }}"
+    state: present
+
+- name: Upd repo links
+  become: true
+  apt:
+    update_cache: yes
+
+- name: Install depencies
+  ansible.builtin.apt:
+    name: "{{ item }}"
+    state: present
+  loop:
+    - apt-transport-https
+    - ca-certificates
+    - curl
+    - software-properties-common
+
+- name: Install docker
+  ansible.builtin.apt:
+    name: "{{ item }}"
+    state: present
+  loop:
+    - docker-ce
```

Comment:
```
failed: [34.118.119.250] (item=docker-ce) => {"ansible_loop_var": "item", "changed": false, "item": "docker-ce", "msg": "No package matching 'docker-ce' is available"}
```
:thinking:

Reply:
Weird, it should have worked. I will check now.

> ```
> failed: [34.118.119.250] (item=docker-ce) => {"ansible_loop_var": "item", "changed": false, "item": "docker-ce", "msg": "No package matching 'docker-ce' is available"}
> ```
> 
> 🤔

Reply:
> ```
> failed: [34.118.119.250] (item=docker-ce) => {"ansible_loop_var": "item", "changed": false, "item": "docker-ce", "msg": "No package matching 'docker-ce' is available"}
> ```
> 
> 🤔

I fixed this.  I wrote repository  wrong. "stablex" instead of "stable"